frontend/src/pages/Admin/Components/AddTeamMember/AddTeamMember.jsx

```diff
@@ -103,6 +104,19 @@ export function AddTeamMember() {
 
   console.log("formerrors: ", formerrors);
 
+  const postTeamMember = (teamMemberData) => {
+    fetch(`${END_POINT}/teamMember/addTeamMember`,{
+      method: 'POST',
+      body: JSON.stringify(teamMemberData),
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': `Bearer ${localStorage.getItem('token')}`,
+      }
+    }).then(response => response.json())
+    .then(data => console.log(data))
+    .catch(error => console.log(error));
```

Comment:
Please add a toast message saying `Member Added Successfully!` in case of success and `something Went wrong!` in case of any error